src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const sumPrice = days * 40;
+  const exchange1 = 20;
+  const exchange2 = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const largeDiscount = 50;
```
1 та 2 - зовсім не зрозуміло, про що йдеться